src/utils/download-file.ts

```diff
@@ -0,0 +1,16 @@
+import { exec } from 'child_process'
+import { Url } from 'url'
+
+export const downloadFile = (url: Url, dest: string) => {
+  return new Promise((resolve, reject) => {
+    const wget = `wget -O "${dest}" "${url.href}"`
```

Comment:
I think we should support resuming from partial downloads (those ZIMs are big):
```suggestion
    const wget = `wget -O -c "${dest}" "${url.href}"`
```

Reply:
The zim file download is happening in the `tools/getzim.sh` script. This download is used by the images in the new homepage, they are downloaded into the /I/m folder so they are available on the homepage.

I will add in the `-c` option anyway, in case this util does get used for larger files in the future.